src/ServiceControl.Monitoring/Infrastructure/HistoryPeriod.cs

```diff
@@ -63,7 +63,9 @@ public override bool Equals(object obj)
             new(TimeSpan.FromMinutes(10), numberOfIntervals: 60, delayedIntervals: 1),
             new(TimeSpan.FromMinutes(15), numberOfIntervals: 60, delayedIntervals: 1),
             new(TimeSpan.FromMinutes(30), numberOfIntervals: 60, delayedIntervals: 1),
-            new(TimeSpan.FromMinutes(60), numberOfIntervals: 60, delayedIntervals: 1)
+            new HistoryPeriod(TimeSpan.FromMinutes(LargestHistoryPeriod), 60, 1)
```

Comment:
either explicitly specify the type of every item, or remove from this one

Reply:
```suggestion
            new(TimeSpan.FromMinutes(LargestHistoryPeriod), 60, 1)
```